Sources/Security.php

```diff
@@ -1013,6 +1025,328 @@ function isAllowedTo($permission, $boards = null)
 		validateSession();
 }
 
+/**
+ * Check the user's character's permissions.
+ * Specifically, which of the user's characters might have a given permission.
+ * If boards is specified, checks those boards instead of the current one.
+ *
+ * @param string|array $permission A single permission to check or an array of permissions to check
+ * @param int|array $boards The ID of a board or an array of board IDs if we want to check board-level permissions
+ * @param int $character A specific character to filter by for easier perms checking.
+ * @return array Which character IDs are permitted to carry out the permission 
```

Comment:
oh, disregard earlier comment. Blech on the similar naming but it's the existing pattern extended

Reply:
Yup, it'll get renamed and revamped sometime.